src/main/java/Duke.java

```diff
@@ -35,7 +81,7 @@ public static void main(String[] args) {
         System.out.println(" What can I do for you?");
         System.out.println("____________________________________________________________");
 
```

Comment:
Should use constants for assigning to these => enhance readability and facilitate coding